web/src/components/cartSummary.js

```diff
@@ -1,5 +1,14 @@
 import React, {Component} from 'react';
+import {gql} from 'apollo-boost';
+import {Mutation} from "react-apollo";
+import {useMutation} from "react-apollo-hooks"
```

Comment:
`useMutation` is only used for a component of function form. We're using a component of class form so, we don't need or use `hooks`